hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/handlers/NodeUpdateHandler.java

```diff
@@ -95,12 +116,16 @@ public void handle(@NonNull final HandleContext handleContext) {
             addressBookValidator.validateGossipEndpoint(op.gossipEndpoint(), nodeConfig);
         if (!op.serviceEndpoint().isEmpty())
             addressBookValidator.validateServiceEndpoint(op.serviceEndpoint(), nodeConfig);
+        if (op.hasAdminKey()) addressBookValidator.validateAdminKeyInHandle(handleContext, op.adminKeyOrThrow());
```

Comment:
We dont need `InHandle` in the method name

Reply:
Removed.